### DISCUSSION TITLE
Windows Terminal Preview v1.18.1462.0

### DISCUSSION THREAD

**Discussion by DHowett** [microsoft/terminal#15474] [Announcements]
Wow, it looks like 1.18 had a bunch of issues. Thanks for bearing with us and using the Preview channel!

* Bug fixes for the new rendering engine:
   * Glyphs that extend above or below the _screen_ (like certain NerdFont varieties) no longer cause rendering failures (#15403)
   * There should be 100% fewer leftover cursor pieces during scrolling (#15427)
   * `DRCS` soft fonts no longer cause crashes (#15419)
   * We've improved scrolling and device invalidation (#15425)
* `wt --help` now no longer emits a crash log after it exits (oops) (#15445)
* In general (outside of the `--help` case,) Terminal should crash even less on exit (#15451)
* We've stamped out some crashes on resize during shutdown (#15457)
* The new environment variable support will now always expand `PATH`, even if it is not a `REG_EXPAND_SZ` (#15444) (thanks @sotteson1!)
* _OpenConsole_: block selection now properly records line breaks (#15423)
* The "Developer Command Prompt" profiles are no longer totally busted (#15439)
* Terminal no longer crashes when you have `focusFollowMouse` mode enabled (#15420)

---

### Asset Hashes

- Microsoft.WindowsTerminalPreview_1.18.1462.0_8wekyb3d8bbwe.msixbundle
   - SHA256 `60BF2D35DAC57DC2878C71C581CC3F000E1875A08AF318615E8FA6DD9288BCDA`
- Microsoft.WindowsTerminalPreview_1.18.1462.0_8wekyb3d8bbwe.msixbundle_Windows10_PreinstallKit.zip
   - SHA256 `17D30178FD2EA39B3DC91E874435025E74C676EF918F5BF79CDF789FE554EC99`
- Microsoft.WindowsTerminalPreview_1.18.1462.0_arm64.zip
   - SHA256 `62FD69046CAF504C456BEB3004CF4E83207D25FD7A03B75D8B2D2A8175630D1D`
- Microsoft.WindowsTerminalPreview_1.18.1462.0_x64.zip
   - SHA256 `C63E631EDBC8211E48EA299A33A5E0ACF713F3115258E8594EDCC402D2F7A093`
- Microsoft.WindowsTerminalPreview_1.18.1462.0_x86.zip
   - SHA256 `B7D47EFA7862800D5CAC6CF75D7EFE2014ED1D0FAAC6B381DEB85F6F186DE915`

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/terminal/releases/tag/v1.18.1462.0'>Windows Terminal Preview v1.18.1462.0</a>.</em>

**Comment by Gianlucas94** [comment]
I can't use the tab tearout function in this version

**Reply by zadjii-msft** [reply]
Are you running the Terminal elevated (as Admin)?

**Reply by Gianlucas94** [reply]
Yes. It worked in a non elevated window.

**Comment by MetAISmith** [comment]


**Reply by MetAISmith** [reply]
- - ~~[]()**@MetAISmith **~~
